### DISCUSSION TITLE
how to change rtl in config

### DISCUSSION THREAD

**Discussion by the-pesar** [quasarframework/quasar#10529] [General - Components / Directives / etc]
**i want change rtl mode in application through user
How to do ?!**

```
build: {
      rtl: true
}
```

**Comment by ontwikkelfabriek** [comment]
@pesarboy 

See this discussion:
https://github.com/quasarframework/quasar/discussions/9342

Here's the demo app with **rtl** switch on **runtime**:
https://qint-demo.pyxo.net/ar

**Reply by Ali-hd** [reply]
> @pesarboy
> 
> See this discussion: #9342
> 
> Here's the demo app with **rtl** switch on **runtime**: https://qint-demo.pyxo.net/ar

Hi, the link doesn't work. Please can you provide another link, I want to see an example with rtl switch on runtime
